src/mlpack/methods/neighbor_search/ns_model_impl.hpp

```diff
@@ -355,7 +394,10 @@ void NSModel<SortPolicy>::Search(arma::mat&& querySet,
   if (randomBasis)
     querySet = q * querySet;
 
-  Log::Info << "Searching for " << k << " nearest neighbors with ";
+  Log::Info << "Searching for " << k;
+  if (Epsilon() != 0)
+    Log::Info << " approximate (e=" << Epsilon() << ")";
```

Comment:
Ooh, I forgot to respond to this, sorry about that. Maybe a clearer way to put it now is something like "searching for K nearest neighbors with a maximum of Y% relative error"; what do you think?